src/nft/index.ts

```diff
@@ -56,7 +58,11 @@ export const mintNFTWithUri = async (testnet: boolean, body: CeloMintErc721 | Et
         case Currency.CELO:
             return sendCeloMinErc721Transaction(testnet, body as CeloMintErc721, provider);
         case Currency.ETH:
-            return sendMintErc721Transaction(body, provider);
+            if (body.hasOwnProperty('authorAddresses')){
```

Comment:
this logic should be for all chains, not only for ETH. just use if body.authorAddress, don't have to use hasOwnProperty